### ISSUE TITLE
Harmonisation and strand flip by PRSice

### ISSUE THREAD

**Issue by jianvhuang** [choishingwan/PRSice#318]
Hi 

I wonder how PRSice deals with inconsistent strand and allele coding between the base GWAS and the target sample. I found a statement from the webpage that PRSsice performs strand flip automatically. Does it also deal with complimentary allele coding (e.g., `EffectAllele.base="C", NonEffectAllele.base="T", A1.target="G", A2.target="A"`)?
In this example, I suppose no flipping of beta is needed.

It will be very helpful if you could elaborate what PRSice checks for this QC step.
Thank you very much.

Regards,
Jian

**Comment by choishingwan** [comment]
Yes.  We check for complement too. We flip by changing the allele dosage.
0 1 2 to 2 1 0

**Comment by jianvhuang** [comment]
Do you think this could be very different from flipping the effect allele and non effect allele (along with the +/- sign of beta) in the base GWAS? 
Could you point out which part of the code in the below file deal with the harmonisation issue? I would like to learn more about this. Thank you very much.
`https://github.com/choishingwan/PRSice/blob/master/PRSice.R`

**Comment by choishingwan** [comment]
It was done within the C++ code. 

The results should be similar. Only reason we ran it the way we did was to align with PLINK.
